### ISSUE TITLE
Black double curly braces in HTML files

### ISSUE THREAD

**Issue by leftbit** [bulenkov/Darcula#16]
AngularJs requires double curly braces. Those are colored black making them very hard to spot. 

![curlybraces](https://cloud.githubusercontent.com/assets/17145796/12923512/4d99d0fe-cf56-11e5-88c9-6aa4fd681fe6.png)

To get a sensible color I changed Options->Fonts&Colors->Syntax->Languabe HTML->Expression Delimiters foreground from "black" to "Inherited". Guess this should be the default.

Nice work BTW. :)

**Comment by AlexFalappa** [comment]
@leftbit I think this is not the correct GitHub project to file your issue.

If you are speaking about NetBeans reopen the issue at https://github.com/Revivius/nb-darcula

**Comment by leftbit** [comment]
Hmmm... they definitely changed from green to black when applying Darcula. So for me it appeared to be related to this project. Cant't remember ever changing that option myself.

**Comment by AlexFalappa** [comment]
@leftbit this project is a Java Swing Look And Feel, you are probably speaking of AngularJS file editing inside the NetBeans IDE after installing the NetBeans Darcula plugin which is based indeed on this look and feel.
The syntax highlighting color scheme is from the NetBeans plugin so I suggested to open the issue in that project.
Anyway I've experienced a similar issue and it got solved by pressing the Reset button in the Fonts&Color tab in the Options dialog. Somehow on first installation the color scheme is not applied thoroughly.
